scripts/crawl_zenodo.sh

```diff
@@ -19,5 +19,5 @@ touch ${TOUCHFILE}
 
 (cd ${BASEDIR} && git pull main master)
 
-docker run --rm -u $UID:$UID -v $HOME:$HOME -e HOME=$HOME -v ${BASEDIR}:/workdir -w /workdir bigdatalabteam/git-annex python3 ./scripts/crawl_zenodo.py --force --verbose -z ${TOKEN_LIST} &>>${LOGFILE}
+docker run --rm -u $UID:$UID -v $HOME:$HOME -e HOME=$HOME -v ${BASEDIR}:/workdir -w /workdir bigdatalabteam/git-annex python3 ./scripts/crawl_zenodo.py --force --verbose -z ${TOKEN_LIST} &>>${LOGFILE} || \rm ${TOUCHFILE}
```

Comment:
I'm unsure how adding `|| \rm ${TOUCHFILE}` does anything since the next line is `\rm ${TOUCHFILE}`

Reply:
You are right, let me remove it